docs/basic_tutorial.rst

```diff
@@ -0,0 +1,216 @@
+********************
+STIPS Basic Tutorial
+********************
+
+This page will take you through the process of creating a basic STIPS observation.  You must have STIPS installed –– if you do not, please see :doc:`installing STIPS <installation>`.
+
+
+Importing STIPS and Checking the STIPS Environment
+--------------------------------------------------
+
+In order to use STIPS, you must have several sets of data files installed (:doc:`installing STIPS <installation>` contains instructions on how to do this).  In order to test your STIPS installation, STIPS includes an environment report utility that shows which version of STIPS you have installed, as well as the versions of the most important support packages that STIPS uses.  When you run the code below, you should get an output set of the following form:
+
+    STIPS Version x.y.z with Data Version x.y.z at /Some/Path/To/stips_data
+
+    STIPS Grid Generated with x.y.z
+
+    Pandeia version a.b.c with Data Version a.b.c. at /Some/Path/To/pandeia_refdata
+
+    Webbpsf Version d.e.f with Data Version d.e.f at /Some/Path/To/webbpsf_data_path
+
+.. code-block:: python
+
+  import stips
+
+  print(stips.__env__report__)
```

Comment:
```suggestion
.. code-block:: python

  import stips
  print(stips.__env__report__)

.. code-block:: text

    STIPS Version x.y.z with Data Version x.y.z at /Some/Path/To/stips_data

    STIPS Grid Generated with x.y.z

    Pandeia version a.b.c with Data Version a.b.c. at /Some/Path/To/pandeia_refdata

    Webbpsf Version d.e.f with Data Version d.e.f at /Some/Path/To/webbpsf_data_path
```
The output should follow the Python code block. The output block should also render as plain text instead of as Python code.